Libplanet/Blocks/Block.cs

```diff
@@ -134,8 +154,10 @@ private Block(RawBlock rb)
                     CultureInfo.InvariantCulture).ToUniversalTime(),
                 rb.Transactions
                     .Select(tx => Transaction<T>.Deserialize(tx.ToArray()))
-                    .ToList()
-            )
+                    .ToList(),
+#pragma warning disable MEN002 // Line is too long
+                rb.Header.ActionsHash.Any() ? new HashDigest<SHA256>(rb.Header.ActionsHash.ToArray()) : (HashDigest<SHA256>?)null)
```

Comment:
Rather than converting `rb.Header.ActionsHash` into an array using `ImmutableArray<byte>.ToArray()` method, how about having an overloaded constructor like `HashDigest<T>(ImmutableArray<byte>)`?

Reply:
How about this...? move to constructor's body
```suggestion
                {
                    ActionsHash = rb.Header.ActionsHash.Any()
                        ? new HashDigest<SHA256>(rb.Header.ActionsHash.ToArray())
                        : (HashDigest<SHA256>?)null)
                }
```

Reply:
What I meant was like:

```csharp
public readonly struct HashDigest<T> : ...
{
    // ...

    public HashDigest(ImmutableArray<byte> hashDigest)  // 👈🏼 Let's having this constructor
    {
        if (hashDigest.Length != Size)
        {
            string message =
                $"HashDigest<{typeof(T).Name}> must be {Size} bytes, " +
                $"but {hashDigest.Length} was given";
            throw new ArgumentOutOfRangeException(
                nameof(hashDigest),
                message
            );
        }

        _byteArray = hashDigest;
    }
}
```

Reply:
Oh, I got it

Reply:
Solved at https://github.com/planetarium/libplanet/pull/935/commits/87bfd7d6da39f561381a32f9206c314b3be8c7cf